frontend/src/components/Board/BuyParts/MpnPopup.module.scss

```diff
@@ -0,0 +1,28 @@
+.MpnPopup {
+  .imageContainer {
+    display: flex;
+    min-width: 160px;
+    min-height: 160px;
+    align-items: center;
+  }
+  .image {
+    align-self: center;
+  }
+  .topAreaContainer {
+    display: flex;
+    align-items: stretch;
+  }
+  .linkContainer {
+    flex-grow: 1;
+    display: flex;
+    justify-content: flex-end;
+    padding-right: 10px;
+    align-items: flex-end;
+    flex-direction: column;
+  }
+  .octopartLinkContainer {
+    font-size: 10px;
+    justify-content: flex-start;
+    align-items: flex-start;
+  }
+}
```

Comment:
Now that these are .module.scss I think we can get rid of the outer classes, i.e. `.MpnPopup` in this case.

Reply:
Resolved via ade6a5d.